scripts/aws/setupdb.sh

```diff
@@ -69,7 +69,10 @@ envdir /etc/mmw.d/env /opt/app/manage.py migrate
 
 function download_and_load {
     for f in "${FILES[@]}"; do
-        curl -s $FILE_HOST/$f | gunzip -q | psql --single-transaction
+        rm -f tmp.sql tmp.sql.gz
```

Comment:
The HUC boundary layers are several hundred MB so piping from curl to gunzip then psql results in "out of memory" exceptions on my machine. And that error only happens after a considerable amount of time. I don't mind reverting this commit.